lib/linter.js

```diff
@@ -51,21 +48,22 @@ class Linter {
   }
 
   loadConfig() {
-    this.config = getConfig(this.options);
+    this.config = getProjectConfig(this.options);
   }
 
-  _defaultSeverityForRule(ruleName, pendingStatus) {
+  _determineSeverityForRule(ruleName, pendingStatus, configRule) {
```

Comment:
FYI - I'd like to see this method go away when we land the overrides support (the logic would move into `getConfigForFile`).

Reply:
yep!

Reply:
need to get this done